src/components/note/note.style.ts

```diff
@@ -37,24 +33,24 @@ const StyledNoteContent = styled.div<{
       margin-top: var(--spacing200);
     `}
   `}
+`;
 
-  ${({ hasInlineControl }) => hasInlineControl && "margin-right: 24px;"}
+const StyledNoteMain = styled.div`
+  display: flex;
+  justify-content: space-between;
+  padding-bottom: 24px;
```

Comment:
Comment: I realise you were just re-organising slightly here so it's no necessarily a change you've made. However, while we're updating these we might as well add any relevant the tokens

```suggestion
  padding-bottom: var(--spacing300);
```